vm/pylib-crate/src/lib.rs

```diff
@@ -2,16 +2,11 @@
 //! common way to use this crate is to just add the `"freeze-stdlib"` feature to `rustpython-vm`,
 //! in order to automatically include the python part of the standard library into the binary.
 
-extern crate self as rustpython_pylib;
-
 pub const LIB_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/Lib");
 
 #[cfg(feature = "compiled-bytecode")]
-use {
-    rustpython_bytecode::bytecode::{self, FrozenModule},
-    std::collections::HashMap,
-};
+use {rustpython_bytecode::FrozenModule, std::collections::HashMap};
 #[cfg(feature = "compiled-bytecode")]
 pub fn frozen_stdlib() -> HashMap<String, FrozenModule> {
-    rustpython_derive::py_freeze!(dir = "Lib", crate_name = "rustpython_pylib")
+    rustpython_derive::py_freeze!(dir = "Lib", crate_name = "rustpython_bytecode")
```

Comment:
is this crate_name chage intended?

Reply:
Yes, I updated compile_bytecode to be a bit more lenient with it. I think it used to require stuff from `rustpython_vm::__exports` but it shouldn't anymore.